bin/coreos-install

```diff
@@ -9,6 +9,18 @@ error_output() {
     echo "Error: return code $? from $BASH_COMMAND" >&2
 }
 
+default_board() {
+    if [[ -e /usr/share/coreos/release ]]; then
+        echo "$(gawk --field-separator '=' '/COREOS_RELEASE_BOARD=/ { print $2 }' /usr/share/coreos/release)"
```

Comment:
This line should be able to just start with `gawk ...` since output does not need to be captured here anymore.